### DISCUSSION TITLE
Green should capture warnings

### DISCUSSION THREAD

**Discussion by henniss** [CleanCut/green#185] [Enhancements]
By default, Python ignores certain warnings to avoid generating excessive noise. Unfortunately, this means that useful warnings such as DeprecationWarnings are likely to be overlooked. 

A test runner is one of the most useful places for them to be visible.  Indeed, the default unittest TestRunner [does show them even if they would be hidden by default](https://docs.python.org/3/library/unittest.html#unittest.TextTestRunner). 

It's possible to get them by running `python -W d -m green`, but this also displays all sorts of other warnings that don't have anything to do with the code under test. 

IMHO Green should follow unittest's lead and enable extra warnings by default.

**Comment by CleanCut** [comment]
Interesting!  Do you have some example tests that ought to generate such warnings?

**Comment by buhtz** [comment]
btw: In my cases warnings are not ignored.
I would assume that the opener missoncfigured its logger instance.

Issue can be closed.
